cmd/scriggo/main.go

```diff
@@ -750,7 +750,6 @@ var stdlibPaths = []string{
 	"io",
 	"io/fs",
 	"io/ioutil",
-	"iter", // Go version 1.23
```

Comment:
The reference to this package should also be removed from the `stdLibPaths` function.

Reply:
Removed in [review: remove "iter" from function stdLibPaths](https://github.com/open2b/scriggo/pull/951/commits/6e655e790714b33a1faff06288f2eb0a5e37ee7b).